prater.tf

```diff
@@ -184,3 +184,17 @@ module "nimbus_nodes_prater_hetzner" {
 
   ips = ["65.21.73.183"]
 }
+
+module "nimbus_nodes_prater_macos" {
+  source = "./modules/hetzner"
```

Comment:
This module should be renamed to something like `dummy-host` or something, since it's being used for non-hetzner host.

Reply:
there is a `mac-stadium` module, i forgot to change it in the previous commit